src/name/verify.rs

```diff
@@ -44,6 +45,51 @@ pub fn verify_cert_dns_name(
     )
 }
 
+pub fn verify_cert_dns_name_or_ip(
+    cert: &crate::EndEntityCert,
+    dns_name_or_ip: DnsNameOrIpRef,
+) -> Result<(), Error> {
+    match dns_name_or_ip {
+        DnsNameOrIpRef::DnsName(dns_name) => verify_cert_dns_name(cert, dns_name),
```

Comment:
Style nit: it would be nice to return for the `DnsName` case, so we can reduce the indentation by two levels for the IP address case.

Reply:
Addressed in https://github.com/rustls/webpki/pull/21.